projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md

```diff
@@ -0,0 +1,29 @@
+---
+remote_theme: nhsx/nhsx-io-theme
+title: Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank
+description: NHSX PhD Internship - Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank
+permalink: /Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/
```

Comment:
Where possible I shortened the slugs, so have done this here as well.

`/Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/`
-> `/transforming-healthcare-data-graph-based-sail/`